playbook/app/pb_kits/playbook/pb_skeleton_loading/docs/_skeleton_loading_filter.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { Button, Card, Filter, Flex, Select, SkeletonLoading, TextInput } from "playbook-ui";
+
+const SortingChangeCallback = (sortOptions) => {
+  alert(JSON.stringify(sortOptions[0]))
+}
+
+const SkeletonLoadingFilter = (props) => {
+  const isLoading = true
+
+  const options = [
+    { value: 'USA' },
+    { value: 'Canada' },
+    { value: 'Brazil' },
+    { value: 'Philippines' },
+    { value: 'A galaxy far far away, like really far away...' },
+  ]
+
+  return (
+    <div>
+      <div>
+        {isLoading ? (
+          <Card 
+              marginBottom="lg" 
+              {...props}
+          >
+              <Flex 
+                  alignItems="center" 
+                  justify="between"
+                  orientation="row"
+              >
+                  <Flex 
+                      alignItems="center" 
+                      justify="start"
+                      orientation="row"
+                  >
+                      <SkeletonLoading
+                          borderRadius="rounded"
+                          height="40px"
+                          marginRight="sm"
+                          width="40px"
+                          {...props}
+                      />
+                      <SkeletonLoading 
+                          height="16px" 
+                          marginRight="md"
+                          width="80px"
+                          {...props}
+                      />
+                  </Flex>
+                  <Flex 
+                      alignItems="center" 
+                      justify="end"
+                      orientation="row"
+                  >
+                      <SkeletonLoading 
+                          height="18px" 
+                          width="120px"
+                          {...props}
+                      />
+                  </Flex>
+              </Flex>
+          </Card>
+        ) : (
+          <Filter
+              filters={{ 'Full Name': 'John Wick' }}
+              marginBottom="lg" 
+              minWidth="375px"
+              results={546}
+              sortOptions={{
+                popularity: 'Popularity',
+                // eslint-disable-next-line
+                manager_title: "Manager\'s Title",
+                // eslint-disable-next-line
+                manager_name: "Manager\'s Name",
```

Comment:
Hey!!! Do we absolutely need the enlist comments? i saw 127 and 128 didn't have them, but the syntax is a little different. just a thought! thanks :)

Reply:
Good idea - modeled these filters off the default/single filter doc example text, definitely not tied to this either way (especially since they never actually render on the page). Can take a look and make it consistent. Thanks!